frontend/src/components/Navs/Nav_v2/index.tsx

```diff
@@ -0,0 +1,135 @@
+import React from "react";
+import {
+  AppBar,
+  Typography,
+  IconButton,
+  Drawer,
+  List,
+  ListItem,
+} from "@material-ui/core";
+import { createStyles, makeStyles, Theme } from "@material-ui/core/styles";
+import Toolbar from "@material-ui/core/Toolbar";
+import { Link, useHistory } from "react-router-dom";
+import MenuIcon from "@material-ui/icons/Menu";
+import { linklist } from "../linklist";
+import useTokenStore from "../../../store/tokenStore";
+import { logout } from "../../../utils/auth/logout";
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      flexGrow: 1,
+    },
+    bg: {
+      background: "transparent",
+    },
+    wrapper: {
+      display: "flex",
+      justifyContent: "space-between",
+      alignItems: "space-between",
+    },
+    menuButton: {
+      marginRight: theme.spacing(2),
+    },
+    logo: {
+      width: "15vh",
+    },
+    other: {
+      color: "white",
+      paddingLeft: 20,
+    },
+    drawerList: {
+      width: 300,
+      background: "#171717",
+      height: "100vh",
+      color: "white",
+    },
+    pseudolink: {
+      cursor: "pointer",
+    },
+  })
+);
+const Nav2: React.FC = () => {
+  const token = useTokenStore((state) => state.token);
+  const setToken = useTokenStore((state) => state.setToken);
+  const classes = useStyles();
```

Comment:
Rename to `styles`

Reply:
I think renaming `classes` to `styles` would be unnecessary as the entire app uses the `classes` keyword for material-ui styles hook.

Reply:
I used `styles` wherever(very few components) I used CSS modules for styling. So, maybe I should change them to `classes`.